app/services/trainees/map_state_from_hesa.rb

```diff
@@ -42,7 +46,11 @@ def trn_received?
     end
 
     def withdrawn?
-      reason_for_leaving.present? && hesa_trainee[:end_date].present?
+      [
+        Hesa::CodeSets::ReasonsForLeavingCourse::MAPPING[TRANSFERRED_TO_ANOTHER_PROVIDER],
+        Hesa::CodeSets::ReasonsForLeavingCourse::MAPPING[DEATH],
+        Hesa::CodeSets::ReasonsForLeavingCourse::MAPPING[FOR_ANOTHER_REASON],
```

Comment:
```suggestion
        WithdrawalReasons::TRANSFERRED_TO_ANOTHER_PROVIDER,
        WithdrawalReasons::DEATH,
        WithdrawalReasons::FOR_ANOTHER_REASON,
```

And then we remove the year-specific HESA codes at the top of this file?

```
TRANSFERRED_TO_ANOTHER_PROVIDER = "03"
DEATH = "05"
FOR_ANOTHER_REASON = "11"
```

Reply:
Oh I didn't realise I could access those values like this! Will change

Reply:
Ah I understand now these should be coming from the initializers! Makes sense